README.md

```diff
@@ -44,3 +44,164 @@ information.
 
 If you pass `--dev`, the `devDependencies` will be checked as well as the
 `dependencies`.
+
+`jsgl` also checks sub-packages for `--local` and `--pr` flags when it
+detects that the repository is a monorepo. It assumes a certain directory
+structure for detecting whether a repository is a monorepo: the top-level
+directory should have the `packages` directory in it and sub-packages must
+exist under that directory. In that case, all the `package.json` files are
+found from sub-packages and `jsgl` checks all of them.
+
+For example, when a directory `foo` is like this:
+```
+foo
+ |
+ +-- packages
+ |    |
+ |    +-- bar
+ |    |    |
+ |    |    +-- package.json
+ |    |    |
+ |    |    +-- ...
+ |    |
+ |    +-- baz
+ |         |
+ |         +-- package.json
+ |         |
+ |         +-- ...
+ |
+ +-- package.json
+ |
+ +-- ...
+```
+, `jsgl` checks all of `foo/package.json`, `foo/packages/bar/package.json`,
+and `foo/packages/baz/package.json`.
+
+## Configurations
+
+You can customize how `jsgl` works with the configuration file, named
+`js-green-licenses.json`. For example, you can specify the license list that
```

Comment:
Could this be a section in the package.json? A top level "js-green-licenses" key?

Reply:
We had a brief discussion about this before. I proposed having a section in package.json and you preferred having a separate JSON file over having a section in package.json because it's not the user of the package needs to see 😄 . I kind of agree with that because I hate to pollute package.json with information that is somewhat-unrelated-to-the-core-functionality.

package.json is simpler though because we can fetch only one file.

Reply:
I have no recollection of that haha :smile:.

There is precent from other modules (e.g. https://www.npmjs.com/package/ava#configuration) to adding config to `package.json`. It is not a lot of information being added, so I don't see a strong reason *not* to put the info directly in there. I don't have really strong feelings though.

Reply:
Yeah, I know about the `ava` section in `package.json`, and that's how I initially got the idea of doing like that.

That said, I have a slight preference in having a separate config file if you don't feel strongly about it. The reason is that that's not really directly related to the development itself. WDYT?

Reply:
Personal opinion, I find putting random config from npm modules in my package.json kind of gross.  I prefer to have a config file just for the module - like a `js-green-licenses.json` or something.  That's just me :)

Reply:
As I said; no strong feelings :).

Reply:
Great. If this looks good, can you LGTM?